docs/content/configuration.md

```diff
@@ -3,99 +3,171 @@ title: Configuration
-between porter and its plugins should be printed when debugging. This can be _very_
-verbose, so it is not turned on by default when debug is true.
+Below is an example configuration file in TOML:
 
-### Output
+```toml
+# ~/.porter/config.toml
 
-`--output` controls the format of the output printed by porter. Each command
-supports a different set of allowed outputs though usually there is some
-combination of: `table`, `json` and `yaml`.
+# Set the default namespace
+namespace = "dev"
 
-### Allow Docker Host Access
+# Include debug logs
+debug = true
 
-`--allow-docker-host-access` controls whether the local Docker daemon
-should be made available to executing bundles. This flag is available for the
-following commands: [install], [upgrade], [invoke] and [uninstall]. When this
-value is set to true, bundles are executed in a privileged container with the
-docker socket mounted. This allows you to use Docker from within your bundle,
-such as `docker push`, `docker-compose`, or docker-in-docker.
+# Include debug logs from the plugins
+debug-plugins = true
 
-🚨 **There are security implications to enabling access! You should trust any
-bundles that you execute with this setting enabled as it gives them elevated 
-access to the host machine.**
+# Default command output to JSON
+output = "json"
 
-⚠️️ This configuration setting is only available when you are in an environment 
-that provides access to the local docker daemon. Therefore it does not work with
-the Azure Cloud Shell driver.
+# Allow all bundles access to the Docker Host
+allow-docker-host-access = true
 
-## Environment Variables
+# Enable experimental v1 features
+experimental = ["build-drivers", "structured-logs"]
 
-Flags have corresponding environment variables that you can use so that you
-don't need to manually set the flag every time. The flag will default to the
-value of the environment variable, when defined.
+# Use Docker buildkit to build the bundle
+build-driver = "buildkit"
 
-`--flag` has a corresponding environment variable of `PORTER_FLAG` and `--another-flag`
-corresponds to the environment variable `PORTER_ANOTHER_FLAG`.
+# Use the storage configuration named devdb
+default-storage = "devdb"
+
+# When default-storage is not set, use the mongodb-docker plugin.
+# This mode does not support additional configuration for the plugin.
+# If the plugin requires configuration, use default-storage and define
+# the configuration in the storage section.
+default-storage-plugin = "mongodb-docker"
+
+# Use the secrets configuration named mysecrets
+default-secrets = "mysecrets"
+
+# When default-secrets is not set, use the kubernetes.secret plugin.
+# This mode does not support additional configuration for the plugin.
+# If the plugin requires configuration, use default-secrets and define
+# the configuration in the secrets section.
+default-secrets-plugin = "kubernetes.secret"
+
+# Defines storage accounts
+[[storage]]
+  # The storage account name
+  name = "cosmos"
```

Comment:
Should/can this be `"devdb"` to match the `default-storage` value above?

Reply:
oops!